v1/artifact.go

```diff
@@ -25,7 +25,7 @@ type ArtifactVersion struct {
 	Tag      string            `json:",omitempty"`
 	Type     string            `json:"artifact_type"`
 	ID       string            `json:"id"`
-	Version  int               `json:"version"`
+	Version  string            `json:"version"`
```

Comment:
I don't think we actually want this. When you use "latest" to retrieve from our API it should still return the latest versions int, right?